diff --git a/openproblems/tasks/denoising/datasets/utils.py b/openproblems/tasks/denoising/datasets/utils.py
index 3a91e9c..d84762e 100644
--- a/openproblems/tasks/denoising/datasets/utils.py
+++ b/openproblems/tasks/denoising/datasets/utils.py
@@ -18,8 +18,8 @@ def split_data(
 
     if scipy.sparse.issparse(X):
         X = np.array(X.todense())
-    if np.allclose(X, X.astype(np.int)):
-        X = X.astype(np.int)
+    if np.allclose(X, X.astype(int)):
+        X = X.astype(int)
     else:
         raise TypeError("Molecular cross-validation requires integer count data.")
 
